### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by tomasmuzas**

### CODE REVIEW

File: BackendApartmentReservation/BackendApartmentReservation.sln [hunk 1]
```diff
@@ -3,7 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
```
Comment (andrejus-kuznecovas) [inline BackendApartmentReservation/BackendApartmentReservation.sln:3]: Is this file needed in this PR? Seems like something that should have been ignored.  Might be wrong.

**Review by andrejus-kuznecovas — APPROVED** [review]

File: BackendApartmentReservation/BackendApartmentReservation.sln (hunk 1 above)
Comment (tomasmuzas) [reply BackendApartmentReservation/BackendApartmentReservation.sln:3]: hmm thats a good question. I think .sln file is the main file, combining all the projects. Its what you open in VS
